addons/nodot/interaction/Teleport3D.gd

```diff
@@ -0,0 +1,88 @@
+## Teleport3D node
+class_name Teleport3D extends Area3D
+
+@export_category("Nodes")
+## Target node for copy the coords
+@export_node_path("Node3D") var Target
```

Comment:
these var names should be lower case. Traditionally only classes start with upper case.